src/styles/index.scss

```diff
@@ -2,2 +1,115 @@
+@import '//fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,100&display=swap';
+
+* {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+
+  &__img {
+    height: 134px;
+    width: 160px;
+    margin-inline: 3px;
```

Comment:
The `margin-inline` property is not part of the Code Style Rules. It's better to use `margin-left` and `margin-right` for consistency and to ensure compatibility with all browsers.